hcs-sxc-java-examples/hcs-sxc-java-settlement-demo/config/docker-compose.sample.yml

```diff
@@ -53,6 +67,45 @@ services:
       - 8084:8084
 
   Erica:
+<<<<<<< HEAD
```

Comment:
Merge gone wrong ?

Reply:
I had disabled the examples. merging bits only until things work.